src/main/webapp/js/common/react/Navbar.js

```diff
@@ -1,36 +1,36 @@
 /**
  * Returns a React navbar component.
  * @param {{}} props The properties of the navbar.
- * @return Returns the navbar.
+ * @return {React.Component} Returns the navbar.
  */
 function Navbar(props) {
   const urls = [
     {
-      href: "/",
-      name: "Home"
+      href: '/',
+      name: 'Home',
     },
     {
-      href: "/projects.html",
-      name: "Projects"
-    }
+      href: '/projects.html',
+      name: 'Projects',
+    },
   ];
 
   return (
     <nav className="navbar navbar-expand-lg navbar-light bg-light">
       <a className="navbar-brand" href="#">Open Sesame</a>
-      <button 
-          className="navbar-toggler" 
-          type="button" 
-          data-toggle="collapse" 
-          data-target="#navbarSupportedContent" 
-          aria-controls="navbarSupportedContent" 
-          aria-expanded="false" 
+      <button
+          className="navbar-toggler"
+          type="button"
```

Comment:
Is the type attribute necessary? Not sure exactly how the style sheets apply to react, but they say to "omit type attributes." https://google.github.io/styleguide/htmlcssguide.html#HTML_Style_Rules

Reply:
Yeah good catch. I took that Navbar boilerplate directly from Bootstrap so it seems like it is missing some of Google's style. Will update!